app/scripts/modules/kubernetes/src/v2/instance/details/details.html

```diff
@@ -50,6 +50,12 @@ <h3 select-on-dbl-click>
         <dd>{{ctrl.instance.kind}}</dd>
       </dl>
     </collapsible-section>
+    <collapsible-section heading="Logs" expanded="true">
+      <li>
```

Comment:
An `li` (list item) is a little odd outside the context of a list (`ul` or `ol`). But whatever works.